core/templates/dev/head/services/UpgradedServices.ts

```diff
@@ -126,6 +129,7 @@ import { WrittenTranslationObjectFactory } from
   'domain/exploration/WrittenTranslationObjectFactory';
 import { WrittenTranslationsObjectFactory } from
   'domain/exploration/WrittenTranslationsObjectFactory';
+import {MessengerService} from './messenger.service';
```

Comment:
Space before and after brackets. Also, avoid relative path naming please.

Reply:
Done